src/JSONDB.php

```diff
@@ -100,7 +107,7 @@ private function check_file()
         } else {
             // Read content of JSON file
             $content = file_get_contents($this->file);
-            $content = json_decode($content, true);
+            $content = json_decode( ($content=="")?'[]':$content, true );
```

Comment:
I do not think this is relevant. We require a JSON file, not an empty text file. So I believe we have to let the user know that this is an invalid JSON file than assume it is supposed to be one.